manual/src/refman/extensions/bindingops.etex

```diff
@@ -92,6 +96,26 @@ let sum3 z1 z2 z3 =
     (fun ((x1, x2), x3) -> x1 + x2 + x3)
 \end{caml_example}
 
+
+\subsection{ss:letops-punning}{Short notation for variable bindings}
+
+(Introduced in 4.13.0)
+
+When the expression being bound is a variable, it can be convenient to
+use the shorthand notation "let+ x in ...", which expands to "let+ x =
+x in ...".  Using this notation, the "sum3" function above can be
+written more concisely as:
+
+\begin{caml_example}{verbatim}
+open ZipSeq
+let sum3 z1 z2 z3 =
+  let+ z1 and+ z2 and+ z3 in
+  z1 + z2 + z3
+\end{caml_example}
+
+This notation is also supported for extension nodes, expanding "let%foo
+x in ..." to "let%foo x = x in ...".
```

Comment:
I would point out explicitly that it is *not* supported for normal lets. (We may even explain that we thought users would find it confusing there.)

Reply:
Good point